app/src/ui/rename-branch/rename-branch-dialog.tsx

```diff
@@ -22,6 +23,7 @@ interface IRenameBranchProps {
 
 interface IRenameBranchState {
   readonly newName: string
+  readonly renaming: boolean
```

Comment:
Could we add a little piece of documentation to this state property? Perhaps something like

```suggestion
  /**
   * Whether or not the component is currently processing the rename.
   * When this is set the dialog will be considered busy and the dialog
   * buttons will be disabled.
   */
  readonly renaming: boolean
```